packages/hebao_v2/contracts/base/libwallet/ERC20Lib.sol

```diff
@@ -115,9 +116,9 @@ library ERC20Lib
         bytes    calldata data
         )
         external
-        returns (bytes memory returnData)
+        returns (bytes32 approvedHash)
```

Comment:
Brecht, I think in callContractWA, we should also return the `returnData` as well:

```
returns (
    bytes32 approvedHash,
    bytes memory returnData
)
```